contrib/mesos/pkg/executor/executor.go

```diff
@@ -390,10 +400,7 @@ func (k *Executor) handleChangedApiserverPod(pod *api.Pod) {
 			oldPod.DeletionTimestamp = pod.DeletionTimestamp
 			oldPod.DeletionGracePeriodSeconds = pod.DeletionGracePeriodSeconds
 
-			k.sendPodUpdate(&kubetypes.PodUpdate{
-				Op:   kubetypes.UPDATE,
-				Pods: []*api.Pod{oldPod},
-			})
+			k.sendPodsSnapshot()
```

Comment:
Can you add a comment that we send the first birth cry here with an empty pod list?